lib/processRunner.js

```diff
@@ -75,46 +46,24 @@ module.exports = function () {
       container.type = 'node'
     }
 
-    if (envFile && !fs.existsSync(envFile)) {
-      return cb('Error: ' + envFile + ' does not exist')
-    }
-
-    if (container.specific && container.specific.environment) {
-      envArgs = generateEnvironment(container.specific.environment)
-    }
-
-    if (envFile) {
-      var envFileData = dotenv.parse(fs.readFileSync(envFile))
-      envArgs += ' ' + generateEnvironment(envFileData || {})
-    }
-
-    if (isWin) {
-      toExec = envArgs + ' ' + cmd
-    } else {
-      toExec = envArgs + 'bash -c "exec ' + cmd + '"'
+    env = Object.create(process.env)
+    if (container.environment) {
+      env = _.merge(env, container.environment)
     }
```

Comment:
process.env is key value only, no need for depth merging, no need to lodash if we're 4+ we can just use Object.assign